### ISSUE TITLE
Error while analyze WCAG using aXe extension

### ISSUE THREAD

**Issue by JakeDinhThanhNguyen** [dequelabs/axe-core#1214]
Hi team,

I'm running the aXe extension to test WCAG for some HTML webpages, but an error occured and stop my test.

Are there anyone has experience on this ? please help.

![image](https://user-images.githubusercontent.com/44628885/47764324-317b5800-dcf7-11e8-9573-5cced5f89ca0.png)

Error: Failed to execute 'querySelectorAll' on 'Document': 'a[href="javascript:clickOnNode("3")"]' is not a valid selector.
    at generateSelector (chrome-extension://iobddmbdndbbbfjopjdgadphaoihpojp/adapter.bundle.js:51951:34)
    at Object.createUniqueSelector [as getSelector] (chrome-extension://iobddmbdndbbbfjopjdgadphaoihpojp/adapter.bundle.js:51990:14)
    at DqElement.get selector [as selector] (chrome-extension://iobddmbdndbbbfjopjdgadphaoihpojp/adapter.bundle.js:51385:48)
    at chrome-extension://iobddmbdndbbbfjopjdgadphaoihpojp/adapter.bundle.js:50378:51
    at Array.map (<anonymous>)
    at chrome-extension://iobddmbdndbbbfjopjdgadphaoihpojp/adapter.bundle.js:50371:47
    at Array.map (<anonymous>)
    at chrome-extension://iobddmbdndbbbfjopjdgadphaoihpojp/adapter.bundle.js:50368:53
    at Array.forEach (<anonymous>)
    at Object../node_modules/axe-core@next/axe.js.helpers.processAggregate (chrome-extension://iobddmbdndbbbfjopjdgadphaoihpojp/adapter.bundle.js:50360:16)

**Comment by WilcoFiers** [comment]
Thanks for reporting the issue. Could you put together a Codepen or something like that, that we can use to reproduce the issue? That would be helpful.

**Comment by marcysutton** [comment]
I can't reproduce it with a demo based on the element source specified above. I'm going to close this issue for lack of an actionable demo. Please provide us with one and we can reopen. https://codepen.io/marcysutton/pen/XyzgZJ
